guides/v2.2/config-guide/cli/config-cli-subcommands-perf-data.md

```diff
@@ -7,543 +7,419 @@ functional_areas:
   - Setup
 ---
 
-## Overview of performance testing data {#config-cli-perf-overview}
+Using the [Magento Performance Toolkit]({{ site.mage2bloburl }}/{{ page.guide_version }}/setup/performance-toolkit), or another tool, for performance testing requires you to generate a large amount of data, including stores, categories, products, and more. You can adjust the amount of data you create using _profiles_, such as small, medium, large, and extra large.
```

Comment:
It's unclear what another tools we are talking about. I guess it's better to eliminate "another tools" from here as we not refer to them.